### DISCUSSION TITLE
Thanks for all the hard work

### DISCUSSION THREAD

**Discussion by tphillip913** [community/community#54770] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I'd like to thank you for all the time and hard work you put into gallery-dl. It has allowed me to grab pictures, save them, and then be able to view them long after the original artist has abandoned or deleted their work. Without your software these artists would be lost to me forever.

So thank you. I do appreciate all you have done and hope you continue to keep the product going.

**Comment by alishahab** [comment]
😍

**Reply by peterkulko** [reply]
Thanks also 👍
